src/stories/components/Card/Card.js

```diff
@@ -9,29 +9,52 @@ import { Button } from '../Button/Button';
  *
  * @component
  * @param {string} modifierClasses Class modifiers of the component.
- * @param {sting} title Text found for the title of the card
- * @param {string} body Text found in the body of the card
- * @param {string} opened Date the experiment will be opened
- * @param {string} admin The admin for the experiment
- * @param {string} researchers The researchers for the experiment
+ * @param {object} experimentInfo The metadata of an experiment
  * @return {object}
  *   <Card
  *      modifierClasses={modifierClasses}
- *      title={text}
- *      text={text}
- *      opened={opened}
- *      admin={admin}
- *      researchesr={researchers} />
+ *      experimentInfo={experimentInfo} />
  * )
  */
 export const Card = ({
   modifierClasses,
-  title,
-  body,
-  opened,
-  admin,
-  researchers,
+  experimentInfo,
 }) => {
+  const {
+    title,
+    body,
+    opened,
+    creator,
+    consentForm,
+    preSurvey,
+    postSurvey,
+    experimentUrl,
+  } = experimentInfo;
+
+  const [showDetails, setShowDetails] = useState(false);
+
+  // To show/hide experiment details
+  const toggleDetails = () => {
+    setShowDetails((prevState) => !prevState);
+  };
+
+  const cardDetails = showDetails ?
+    <React.Fragment>
+      <hr/>
+      <p>Consent form Qualtrics survey:</p>
```

Comment:
This is simply a styling preferences, but I'd say it's better to use <h5>Consent form Qualtrics survey:</h5>.
Similarly on line 47, 50, and 53.